projects/imagenet/experiments/base.py

```diff
@@ -25,10 +30,14 @@
 import torch
 
 import nupic.research.frameworks.pytorch.models.resnets
+from nupic.research.frameworks.pytorch.imagenet import ImagenetExperiment, mixins
+
+
+class RezeroedKWinnersImagenetExperiment(mixins.RezeroWeights,
+                                         mixins.UpdateBoostStrength,
+                                         ImagenetExperiment):
+    pass
```

Comment:
If think this class belongs to `nupic.research.frameworks.pytorch.imagenet` package so it can be easily reused across projects